sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/init.sql

```diff
@@ -74,9 +84,12 @@ first_session_ping AS (
     ] AS adjust_creative
   FROM
     `moz-fx-data-shared-prod.fenix.first_session` AS fenix_first_session
+  INNER JOIN
+    first_session_ping_min_seq
+  ON
+    client_info.client_id = first_session_ping_min_seq.client_id
   WHERE
     DATE(submission_timestamp) >= '2019-01-01'
-    AND ping_info.seq = 0 -- Pings are sent in sequence, this guarantees that the first one is returned.
```

Comment:
An option for not missing seq = NULL could be adding here:

`ping.info.seq IN (subset) OR NULL`